triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/ServiceTest.scala

```diff
@@ -62,6 +66,22 @@ class ServiceTest extends AsyncFlatSpec with Eventually with Matchers {
   implicit val esf: ExecutionSequencerFactory = new AkkaExecutionSequencerPool(testId)(system)
   implicit val ec: ExecutionContext = system.dispatcher
 
+  protected def jwt(party: String): Jwt = {
+    println("test id" + testId)
+    val decodedJwt = DecodedJwt(
```

Comment:
Why embed this as a string instead of using `AuthServiceJWTPayload`? See https://github.com/digital-asset/daml/blob/master/ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/sandbox/services/SandboxFixtureWithAuth.scala#L60 for an example of the latter.

Reply:
I just copied the json-api code. I'll have a look!